frontend/packages/client/src/components/ProposalInformation/ProposalInformation.js

```diff
@@ -0,0 +1,177 @@
+      if (windowHeight > navbarHeight + infoPanelHeightSize) {
+        // user scrolled down and panel is next to navbar
+        // adding 4px so the nav bar sticks more smoothly
+        if (top < navbarHeight + 4 && !fixedStyle) {
+          // save reference where the element needs to go back
+          if (!topRef.current.returnTop) {
+            topRef.current.pointStatic = winScroll;
+          }
+          setFixedStyle({
+            className: ' is-panel-fixed',
+            // use width of parent component
+            style: {
+              width,
+              top: navbarHeight,
+            },
+          });
+        }
+      }
+
+      if (
+        fixedStyle &&
+        topRef.current.pointStatic &&
+        topRef.current.pointStatic > winScroll
+      ) {
+        topRef.current.pointStatic = null;
+        setFixedStyle(null);
+      }
+    }
+  }, [scroll, fixedStyle, windowHeight, navbarHeight]);
+
+  // this effect watches for window width resizes to change info panel width
+  useEffect(() => {
+    if (parentRef?.current) {
+      const { width } = parentRef?.current.getBoundingClientRect() || {};
+
+      // window was resized then change width of the component if fixed style was applied
+      if (fixedStyle && fixedStyle.windowWidth !== windowWidth) {
+        setFixedStyle((state) => ({
+          ...state,
+          windowWidth,
+          style: { ...(state?.style ?? {}), width },
+        }));
+      }
+    }
+  }, [fixedStyle, windowWidth, navbarHeight]);
+
+  function handleScroll() {
+    setScroll(document.body.scrollTop || document.documentElement.scrollTop);
+  }
+
+  // this effect watches for window scrolling
+  useEffect(() => {
+    document.addEventListener('scroll', handleScroll);
+    return () => document.removeEventListener('scroll', handleScroll);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div ref={parentRef}>
+      <div
+        className={`${fixedStyle?.className ?? ''}`}
+        ref={ref}
+        style={fixedStyle?.style || {}}
+      >
+        <div className="mb-5">
+          <ResultsPanel
+            results={votingResults?.results}
+            endTime={endTime}
+            computedStatus={computedStatus}
+          />
+        </div>
+        <Information
+          creatorAddr={creatorAddr}
+          strategyName={strategyName}
+          isCoreCreator={isCoreCreator}
+          ipfs={ipfs}
+          ipfsUrl={ipfsUrl}
+          startTime={startTime}
+          endTime={endTime}
+          communityName={communityName}
+          communityLogo={communityLogo}
+          communitySlug={communitySlug}
+          communityId={communityId}
+          tokenName={tokenName}
+          maxWeight={maxWeight}
+          minBalance={minBalance}
+          openStrategyModal={openStrategyModal}
+        />
+      </div>
+    </div>
+  );
+};
+
+export default ProposalInformation;
```

Comment:
This was an existing component, moved to another location and extracted components from it